include/pmacc/particles/ParticlesBase.hpp

```diff
@@ -172,6 +172,11 @@ class ParticlesBase : public SimulationFieldHelper<T_MappingDesc>
     /** copy gard particles to intermediate exchange buffer
      *
      * Copy all particles from the guard of a direction to the device exchange buffer.
+     *  @warning This method resets the number of particles in the processed supercells even
+     * if there are particles left in the supercell and not fulfill the rule that the
+     * last frame is contiguous filled.
+     * Call KernelFillGaps afterwards if you need a valid number of particles
+     * and a contiguous filled last frame.
```

Comment:
I think `contiguous filled last frame` should be fixed to either `contiguous last frame` or `contiguously filled last frame`.